### PR TITLE
added font awsome facebook icon

### PR DESCRIPTION
**Pull request by CodeRhyno**
Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [x] code is in uniquely-named feature branch and has no merge conflicts
* [x] PR is descriptively titled
* [x] PR body includes `fixes #0000`-style reference to original issue #
* [x] ask `@publiclab/reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software

We have a loose schedule of reviewing and pulling in changes every Tuesday and Friday, and publishing changes on Fridays. 

Thanks!

### CODE REVIEW

**Comment by namangupta01** [comment]
@CodeRhyno can you please add screenshot?

**Comment by namangupta01** [comment]
And also it would be good if you link this pr with the issue like Fixes#1234
Thanks

**Comment by CodeRhyno** [comment]
This is with reference to #2987
screen shot 2018-07-06 at 12 42 04 pm
screen shot 2018-07-06 at 12 44 13 pm
screen shot 2018-07-06 at 12 44 58 pm
screen shot 2018-07-06 at 12 45 40 pm

File: app/views/layouts/_header.html.erb [hunk 1]
```diff
@@ -186,8 +186,10 @@
                   <% if Rails.env == "production"%>
                    <h5>Log In with</h5>
                     <div class="loginbutton" style="float:left">
-                      <a href="/auth/google_oauth2" id="connect-google"><span class="btn btn-default" style="margin-right:2px"><i class="fa fa-google" style="font-size:20px;color:red;"></i></span></div></a>
-                      <a href="/auth/github" id="connect-github"><span class="btn btn-default" style="margin-left:2px"><i class="fa fa-github" style="font-size:20px;color:green;"></i></span></div></a>
+                      <a href="/auth/google_oauth2" id="connect-google"><span class="btn btn-default" style="margin-right:2px"><i class="fa fa-google" style="font-size:20px;color:red;"></i></span></a>
+                      <a href="/auth/github" id="connect-github"><span class="btn btn-default" style="margin-left:2px"><i class="fa fa-github" style="font-size:20px;color:green;"></i></span></a>
+                      <a href="/auth/facebook" id="connect-facebook"><span class="btn btn-default" style="margin-left:2px"><i class="fa fa-facebook" style="font-size:20px;color:blue;"></i></span></a>
+                    </div>
```
Comment (namangupta01) [inline app/views/layouts/_header.html.erb]: i guess this div is extra?

**Comment by namangupta01** [comment]
Good work! It is now good to merge. 🎉

**Comment by CodeRhyno** [comment]
Thanks @namangupta01. 😄

**Comment by jywarren** [comment]
Wow, this is awesome! Thank you!!!

**Comment by jywarren** [comment]
If you're looking for a new project, we'd love to continue collaborating! You can check out:

https://github.com/publiclab/plots2/labels/help-wanted

**Pull request merged**